### PR TITLE
Add stripWhitespace

### PR DESCRIPTION
**Pull request by vikpe**
Strip all whitespace characters (similar to `collapseWhitespace`).

### CODE REVIEW

File: src/Stringy.php [hunk 1]
```diff
@@ -1250,6 +1250,18 @@ public function split($pattern, $limit = null)
...
     /**
+     * Strip all whitespace characters. This includes tabs and newline
+     * characters, as well as multibyte whitespace such as the thin space
+     * and ideographic space.
+     *
+     * @return Stringy Object with whitespace stripped
+     */
+    public function stripWhitespace()
+    {
+        return $this->regexReplace('[[:space:]]+', '');
```
Comment (danielstjules) [inline src/Stringy.php]: Thanks for the PR! I'm a bit on the fence with whether or not it's necessary. Is it that common, and that much more convenient over writing `$str->regexReplace('[[:space:]]+', '');`

Comment (vikpe) [inline src/Stringy.php]: My two cents

1. **It fits well into the public interface**, there are several other whitespace-related methods available  (`collapseWhitespace, tidy, toSpaces, toTabs, trim, trimLeft, trimRight`).

2. **Code readability** (compared to using a callback method or iteratation), eg:
```php
$formatted_numbers = array_map('\Stringy\StaticStringy::stripWhitespace', $numbers);
```

As my favorite author would put it (_Steve McConnell, Code Complete_):
> Treat yourself to the highest possible level of abstraction

3. **Regarding usage** (from Google):
* "php strip all whitespace" (5 220 000 results)
* "php collapse whitespace" (234 000 results)

It's your repo, your call! :smile:

Comment (danielstjules) [inline src/Stringy.php]: Well argued :)

Comment (vikpe) [inline src/Stringy.php]: Thank you very much! I'll make another PR for the documentation.

**Pull request merged**